From bfbd7b598bab389183c6d8d4852555c41e99b23e Mon Sep 17 00:00:00 2001
From: Toomas Soome <tsoome@me.com>
Date: Sun, 4 Mar 2018 08:27:35 +0200
Subject: [PATCH] 9173 smbsrv: we do not build all Reviewed by: Andy Fiddaman
 <omnios@citrus-it.co.uk> Reviewed by: Yuri Pankov <yuripv@yuripv.net>
 Approved by: Gordon Ross <gwr@nexenta.com>

---
 usr/src/lib/smbsrv/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/usr/src/lib/smbsrv/Makefile b/usr/src/lib/smbsrv/Makefile
index 9ba411e8ce..4213133a56 100644
--- a/usr/src/lib/smbsrv/Makefile
+++ b/usr/src/lib/smbsrv/Makefile
@@ -34,9 +34,9 @@ SUBDIRS = \
 	libsmbns \
 	libsmbrp
 
+include ./Makefile.subdirs
+
 libmlrpc: libsmb
 libsmbns: libsmb
 libmlsvc: libsmb libmlrpc libsmbns
 libfksmbsrv: libsmb
-
-include ./Makefile.subdirs
-- 
2.21.0

